### PR TITLE
[main.js] getHTML: add aria-label to correct childnode

### PR DESCRIPTION
**Pull request by pkra**
Resolves #289

### CODE REVIEW

**Comment by pkra** [comment]
Tests fixed.

File: lib/main.js [hunk 1]
```diff
@@ -569,9 +569,12 @@ function GetHTML(result) {
 
   // add speech text if there isn't one
   if (data.speakText){
-    for (var i=0, m=html.childNodes.length; i < m; i++)
-      html.childNodes[i].setAttribute("aria-hidden",true);
-    if (!html.getAttribute("aria-label")) html.setAttribute("aria-label",result.speakText);
+    var labelTarget = html.querySelector('.mjx-math');
+    for (var i=0, m=labelTarget.childNodes.length; i < m; i++)
```
Comment (zorkow) [inline lib/main.js]: for (var i = 0, child; child = childNode[i]; i++) {
...
}

**Comment by pkra** [comment]
Question: why was the label added twice? shouldn't speakText stop when there's alttext from mathml?

**Comment by pkra** [comment]
We do add speakText twice if mml with alttext is the input, which technically isn't necessary. I'm ok with it as is. 

A better solution would be nice, i.e., updating the internal format and leaving it to mathjax. But that would require some refactoring of the chain of events, e.g., splitting the mathjax processing so we can update the internal format; that seems out of scope here.

File: lib/main.js (hunk 1 above)
Comment (pkra) [inline lib/main.js]: PTAL.

**Comment by pkra** [comment]
@zorkow ptal

**Comment by pkra** [comment]
Ok per F2F
